products/knownprojects/bash/01_dataloading.sh

```diff
@@ -3,16 +3,22 @@ source ../../bash/utils.sh
 set_error_traps
 max_bg_procs 5
 
+if [ -n "${BUILD_ENGINE_SCHEMA}" ]; then
+    echo "Dropping and creating build schema '$BUILD_ENGINE_SCHEMA'"
+    run_sql_command "DROP SCHEMA IF EXISTS ${BUILD_ENGINE_SCHEMA} CASCADE;"
+    run_sql_command "VACUUM (ANALYZE);"
```

Comment:
I don't think we need the analyze here as we don't need to compute any new statistics on tables, but I would say we should do `VACUUM FULL` to make sure we're freeing up storage space on the db

Reply:
sure I'll drop ANALYZE

unfortunately I was using FULL at first and was getting timeouts